### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by jaemannyeh**
I have developed the repository available at https://github.com/jaemannyeh/SharkJ1939. Could you kindly remove this repository, please?

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @jaemannyeh. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/02/21 16:07:19 Loaded 6679 libraries from DB
Removing SharkJ1939
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/jaemannyeh/SharkJ1939 not present
Success!
```

**Pull request merged**
